platform/android/java/src/org/godotengine/godot/Godot.java

```diff
@@ -100,6 +100,8 @@
 	static final int REQUEST_RECORD_AUDIO_PERMISSION = 1;
 	static final int REQUEST_CAMERA_PERMISSION = 2;
 	static final int REQUEST_VIBRATE_PERMISSION = 3;
+	static final int REQUEST_READ_EXTERNAL_STORAGE_PERMISSION = 3;
```

Comment:
Shouldn't these be 4 and 5?

Reply:
I fixed it while resolving conflict. I will fix it.